.github/workflows/terraform-test.yml

```diff
@@ -0,0 +1,28 @@
+name: 'Terraform Module Test'
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  tf-test:
+    runs-on: ubuntu-latest
+    permissions: {}
+    steps:
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
+      - uses: hashicorp/setup-terraform@651471c36a6092792c552e8b1bef71e592b462d8
+        with:
+          terraform_version: 1.9.2
+          terraform_wrapper: false
+      - name: terraform_test
+        env:
+            TF_VAR_org_id: ${{ vars.MONGODB_ATLAS_ORG_ID }}
```

Comment:
why isn't this var all uppercase?  like TF_VAR_ORG_ID?   also we normally use MONGODB_ATLAS_ORG_ID, e.g.: https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/.github/workflows/acceptance-tests-runner.yml#L143

Reply:
In this case I was following the Terraform convention when defining environment variables names, which is TF_VAR_{name of the variable}. https://developer.hashicorp.com/terraform/cli/config/environment-variables#tf_var_name

Reply:
but we're not using for the others like MONGODB_ATLAS_BASE_URL, not sure why the difference

Reply:
oh, is it because the org variable is an input param module but the others are env. vars to be read by the resources?

Reply:
in that case, ok to resolve this, thanks!